pepper-apis/src/main/java/org/broadinstitute/ddp/export/collectors/ActivityResponseCollector.java

```diff
@@ -63,14 +66,18 @@
     private MailingAddressFormatter addressFmt = new MailingAddressFormatter();
     private MedicalProviderFormatter providerFmt = new MedicalProviderFormatter();
 
+    private final AddressService addressService;
+
     public ActivityResponseCollector(ActivityDef definition) {
         this.definition = definition;
+        this.addressService = new AddressService();
```

Comment:
`AddressService` is not used in this collector.

Reply:
Removed